src/main/java/org/prebid/server/bidder/model/BidderError.java

```diff
@@ -67,9 +69,19 @@ public static BidderError timeout(String message) {
          * generate an error explaining the deficiency. Otherwise it will be extremely difficult to debug the reason
          * why a bidder is not bidding.
          */
-        failed_to_request_bids,
+        failed_to_request_bids(4),
+
+        timeout(1),
+        generic(999);
+
+        private final Integer errorCode;
+
+        Type(final Integer errorCode) {
+            this.errorCode = errorCode;
+        }
 
-        timeout,
-        generic
+        public Integer getErrorCode() {
```

Comment:
`Error` in method and field name looks redundant as it is evident from the context it is all about error